baseapp_chats/graphql/mutations.py

```
@@ -404,6 +384,164 @@ def mutate_and_get_payload(
+            return ChatRoomsAddParticipant(
+                errors=[
+                    ErrorType(
+                        field="profile_id",
+                        messages=[_("You don't have permission to act as this profile")],
+                    )
+                ]
+            )
+
+        try:
+            participant_profile = get_obj_from_relay_id(info, participant_profile_id)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)